tests/core/test_policies.py

```diff
@@ -379,28 +420,42 @@ def create_policy(
         resource: Resource,
         execution_context: ExecutionContext,
         config: Optional[Dict[Text, Any]] = None,
-    ) -> Policy:
-        max_history = None
-        if isinstance(featurizer, MaxHistoryTrackerFeaturizer):
-            max_history = featurizer.max_history
-        return MemoizationPolicy(priority=priority, max_history=max_history)
+    ) -> EitherPolicy:
+        config = config or MemoizationPolicyGraphComponent.get_default_config()
+        config[POLICY_PRIORITY] = priority
+        return MemoizationPolicyGraphComponent(
+            config, model_storage, resource, execution_context, featurizer
+        )
 
-    def test_featurizer(self, trained_policy: Policy, tmp_path: Path):
+    def test_featurizer(
+        self,
+        trained_policy: EitherPolicy,
+        resource: Resource,
+        model_storage: ModelStorage,
+        tmp_path: Path,
```

Comment:
no longer needed
```suggestion
```

Reply:
It's needed as we are subclassing PolicyTestCollection. But I actually just checked that only Memoization and Ted extend that test suite - So basically I could remove all of that old code non-graph code right?

Reply:
indeed 👍🏻

Reply:
Ah yikes, now I did a ton of changes but actually UnexpecTED extends TestTedPolicy, but isn't converted yet -.-

Reply:
ah wait seems that was finished an hour ago